File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_ANDROID/SAMPLE_ANDROID/payapp_android_sample_v3/AppLink/src/main/java/com/udid/applink/model/PACashReceiptResponseData.java
package com.udid.applink.model;

import android.content.Intent;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * 영수증 발행 결과값
 * Create by 김진원
 */
@Getter
@Setter
@ToString(callSuper = true)
public class PACashReceiptResponseData extends PACommonResponseData{

    /**
     * 현금영수증 발행 번호
     */
    String cashstno;

    /**
     * 승인 번호
     */
    String receiptno;

    /**
     * 영수증 url
     */
    String cashsturl;

    /**
     * 금액
     */
    String price;

    /**
     * 임의 변수 1, 2
     */
    String var1;
    String var2;

    public PACashReceiptResponseData(Intent data) {
        super(data);
        cashstno = data.getData().getQueryParameter("cashstno");
        receiptno = data.getData().getQueryParameter("receiptno");
        cashsturl = data.getData().getQueryParameter("cashsturl");
        price = data.getData().getQueryParameter("price");
        var1 = data.getData().getQueryParameter("var1");
        var2 = data.getData().getQueryParameter("var2");
    }




}

File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_IOS/SAMPLE_IOS/payapp_ios_sample_v3/UniversalLink/model/PACashReceiptResponseData.swift
//
//  PACashReceiptResponseData.swift
//  Payapp_iOS_Sample_V3
//
//  Created by 조상현 on 2023/08/22.
//
 
class PACashReceiptResponseData: PACommonResponseData {
     
    /**
     발행번호
     */
    var cashstno: String?
    /**
     영수증번호
     */
    var receiptno: String?
    /**
     영수증 url
     */
    var cashsturl: String?
    /**
     금액
     */
    var price: String?
    /**
     임의 변수 1, 2
     */
    var var1: String?
    var var2: String?
    
    
    override init(_ data:[String:Any]) {
        super.init(data)
        
        cashstno                = data["cashstno"] as? String
        receiptno               = data["receiptno"] as? String
        cashsturl               = data["cashsturl"] as? String
        price                   = data["price"] as? String
        var1                    = data["var1"] as? String
        var2                    = data["var2"] as? String
    }
    
    func toString() -> String {
        var str = "state:\(state ?? ""), "
        str += "errorMessage:\(errorMessage ?? ""), "
        str += "cashstno:\(cashstno ?? ""), "
        str += "receiptno:\(receiptno ?? ""), "
        str += "cashsturl:\(cashsturl ?? ""), "
        str += "price:\(price ?? ""), "
        str += "var1:\(var1 ?? ""), "
        str += "var2:\(var2 ?? "")"
        return str
    }
}
